src/python/pants/option/global_options.py

```diff
@@ -698,7 +698,7 @@ def register_bootstrap_options(cls, register):
             help=(
                 "Read additional specs (target addresses, files, and/or globs), one per line, "
                 "from these files."
-            ),
+            ), # noqa: PANTSBIN
```

Comment:
Why this one?

Reply:
This one is my mouse which accidentally middle-clicks during a scroll and pasts text :facepalm: